src/main/java/world/bentobox/bentobox/api/commands/island/team/IslandTeamPromoteCommand.java

```diff
@@ -52,6 +52,11 @@ public boolean execute(User user, String label, List<String> args) {
             user.sendMessage("general.errors.unknown-player", TextVariables.NAME, args.get(0));
             return true;
         }
+        // Check if the user is not trying to promote/ demote himself
+        if (target == user) {
+            user.sendMessage("demote.errors.cant-demote-yourself");
```

Comment:
Wrong message string.

It should be "commands.island.team.demote.errors.cant-demote-yourself"

Reply:
Thanks @BONNe ! #835 changes it